operator/pkg/preflight/upgrade_test.go

```diff
@@ -51,15 +51,15 @@ type FakeRepo struct {
 
 var _ manifest.Repository = &FakeRepo{}
 
-func (r *FakeRepo) LoadManifest(ctx context.Context, component string, id string, o declarative.DeclarativeObject) (map[string]string, error) {
+func (r *FakeRepo) LoadManifest(_ context.Context, _ string, _ string, _ declarative.DeclarativeObject) (map[string]string, error) {
```

Comment:
Nit: These seem a little overzealous when we're implementing an interface, but /shrug